### DISCUSSION TITLE
ai.djl.engine.EngineException: PytorchStreamReader failed locating file constants.pkl: file not found

### DISCUSSION THREAD

**Discussion by LiorXu** [deepjavalibrary/djl#1946] [Q&A] [answered]
### PytorchStreamReader failed locating file constants.pkl: file not found
l  ensure  that  the files are existing, but the app can't find them. How can l resolve this ? Please. 

![Screenshot 2022-08-25 000559](https://user-images.githubusercontent.com/60100853/186468369-beb31da8-45f4-45df-8ef3-42cf2c2999fb.png)

**Comment by frankfliu** [comment, marked as answer]
Did you trace your model using `jit.trace()`?

See: https://docs.djl.ai/master/docs/pytorch/how_to_convert_your_model_to_torchscript.html

**Reply by LiorXu** [reply]
Thank you very much. My model was use torch.save(). Now l have changed it , this application is successful. Thank your help again.
